addons/common/scripts/checkVersionNumber.sqf

```diff
@@ -1,13 +1,13 @@
 // by commy2
 #include "script_component.hpp"
 
-private _files = [];
-
-{
-    if (_x find "a3_" != 0 && {_x find "ace_" != 0} && {!(toLower _x in (missionNamespace getVariable ["ACE_Version_Whitelist", []]))}) then {
-        _files pushBack _x;
-    };
-} forEach activatedAddons;
+private _aceWhitelist = missionNamespace getVariable ["ACE_Version_Whitelist", []];
+private _files = CBA_common_addons select {
+    private _x = toLower _x;
+    ((_x select [0,3]) != "a3_") &&
+    {(_x select [0,4]) != "ace_"} && 
```

Comment:
too many parenthesii

Reply:
Make suggestions and I'll click on the commit suggestion button.

Reply:
```suggestion
    {_x select [0,4] != "ace_"} && 
```
For @commy2.